tests/Fixtures/App/DataFixtures/Phpcr/LoadTreeData.php

```diff
@@ -20,7 +20,7 @@
 
 class LoadTreeData implements FixtureInterface
 {
-    public function load(ObjectManager $manager)
+    public function load(ObjectManager $manager): void
```

Comment:
Isn't this a BC-break for extending classes?

Reply:
its a fixture